kids-learning-app/assets/js/app.js

```diff
@@ -0,0 +1,260 @@
+        shapes.setAttribute("class","shapes-container");
+
+        let title = components.titleDiv();
+        title.innerText = "1. Choose a Shape"
+
+        let nextButton = components.nextButton();
+        nextButton.innerText = "NEXT";
+        nextButton.addEventListener("click",() => {
+            lsobj.page = 2;
+            localStorage.setItem("lsShape",JSON.stringify(lsobj));
+            return loadBodyPage(); 
+        });
+        let circle = components.circleButton();
+        circle.append(components.checkIcon("chkcircle"));
+        
+        let triangle = components.triangleButton();
+        triangle.append(components.checkIcon("chktriangle"));
+        
+        let square = components.squareButton();
+        square.append(components.checkIcon("chksquare"));
+
+        shapes.append(circle,triangle,square);
+        body.append(title,shapes);
+
+        document.querySelectorAll(".fa-check").forEach(el => el.style.display = "none");
+        
+        if(lsobj.isSelected != false){
+            body.append(nextButton);
+        }
+
+        shapes.addEventListener("click", (event) => {
+            document.querySelectorAll(".fa-check").forEach(el => el.style.display = "none");
+            if(event.target.className.indexOf("chk") >= 0){
+                lsobj.shape = event.target.className.split(" ")[2].slice(3);
+                console.log(lsobj.shape);
+            }else{
+                lsobj.shape = event.target.className;
+                console.log(lsobj.shape);
+            }
+            document.querySelector(".chk" + lsobj.shape).style.display = "inline";
+            lsobj.isSelected = true;
+            localStorage.setItem("lsShape",JSON.stringify(lsobj));
+            body.append(nextButton);
+        });
+    },
+
+    // returns the dimension page(2nd page)
+    dimension:() => {
+        var lsobj = JSON.parse(localStorage.getItem("lsShape"));
+        var body = components.bodyContainer();
+        var title = components.titleDiv();
+        var inputBox = components.inputBox();
+        var calculate = components.nextButton();
+        calculate.innerText = "CALCULATE";
+
+        switch(lsobj.shape){
+            case('circle'):
+                title.innerText = "2. Enter Radius";
+                inputBox.setAttribute("placeholder","Enter Radius...");
+                break;
+            case('triangle'):
+                title.innerText = "2. Enter Side (Base & Height)";
+                inputBox.setAttribute("placeholder","Enter base height...");
+                break;
+            case('square'):
+                title.innerText = "2. Enter Side";
+                inputBox.setAttribute("placeholder","Enter side...");
+                break;
+        }
+        
+        calculate.addEventListener("click",() => {
+            var inp = parseInt(inputBox.value);
+            if(inp <= 0 || inputBox.value === ''){
+                alert("Enter proper dimension.");
```

Comment:
Good that the validation is taken care of, the user may not be aware of the value range, a detailed note would help.